File: src/main/java/com/leijendary/spring/boot/template/model/SampleTableTranslations.java
package com.leijendary.spring.boot.template.model;

import javax.persistence.Embeddable;

import com.leijendary.spring.boot.template.core.model.LocaleModel;

import lombok.Data;
import lombok.EqualsAndHashCode;

@Data
@EqualsAndHashCode(callSuper = true)
@Embeddable
public class SampleTableTranslations extends LocaleModel {

    private String name;
    private String description;
}

File: src/main/kotlin/com/leijendary/spring/boot/template/model/SampleTableTranslations.kt
package com.leijendary.spring.boot.template.model

import com.leijendary.spring.boot.template.core.model.LocaleModel
import javax.persistence.Embeddable

@Embeddable
data class SampleTableTranslations(
    var name: String,
    var description: String,
    override var language: String,
    override var ordinal: Int
) : LocaleModel(language, ordinal)